src/scripts/components/structures/header/MainHeader.js

```diff
@@ -184,7 +209,7 @@ class MainHeader extends Component<Props, State> {
         {this.props.children}
         <HeaderWrapper open={this.state.navOpen}>
           <LogoWrapper>
-            <MainHeaderLogo />
+            <MainHeaderLogo clickOnLogo={this.closeNavAndUserNav} />
```

Comment:
What is `nav and user nav`? Can we have a more descriptive name for `nav`?

Reply:
We have an `UserNav` component, and nav is our main navigation.
What name that you suggest?

Reply:
Done!